kubernetes/cache/kube_cache.go

```diff
@@ -443,6 +443,59 @@ func (c *kubeCache) createKubernetesInformers(namespace string) informers.Shared
 	if namespace != "" {
 		opts = append(opts, informers.WithNamespace(namespace))
 	}
+	opts = append(
+		opts,
+		informers.WithTransform(func(obj interface{}) (interface{}, error) {
+			if pod, ok := obj.(*core_v1.Pod); ok {
```

Comment:
In the future we can add one of these transform funcs for each of the types that kiali watches e.g. deployments, statefulsets, configmaps etc.

Turning this into a `switch` statement would make it easier to add those:
```
switch o := obj.(type) {
case *core_v1.pod:
  trimmedPod: &core_v1.Pod{
    ObjectMeta: metav1.ObjectMeta{
      name: o.Name
   ...
case *core_v1.Service:
...

```

Reply:
You might be able to use something like this to trim metadata for all objects at the top or bottom of this function if it's common to all objects: https://github.com/openshift/ovn-kubernetes/blob/8f1dcb6a878fcd22b4cf2727abeba7f7c69ba64b/go-controller/pkg/factory/factory.go#L212-L214

Reply:
yeah good ideas, have applied